### PR TITLE
New package: f3d-2.1.1

### PR DESCRIPTION
**Pull request by Jipok**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)
- I cant built this PR locally for armv6l,aarch64-musl architectures:
```
=> ERROR: hdf5-devel-1.10.5_2: cannot be cross compiled...
=> ERROR: hdf5-devel-1.10.5_2: https://portal.hdfgroup.org/pages/viewpage.action?pageId=48808266
```

f3d is like feh but for 3d models. Very convenient for quickly viewing models. Especially if you have never worked with such programs before.

### CODE REVIEW

**Comment by Jipok** [comment]
Resolves https://github.com/void-linux/void-packages/issues/43629

File: srcpkgs/f3d/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=f3d
+version=2.1.0
+revision=1
+build_style=cmake
+makedepends="vtk-devel"
+depends="vtk"
+short_desc="Fast and minimalist 3D viewer"
+maintainer="Jipok <void@jipok.ru>"
+license="BSD-3-Clause"
+homepage="htpps://f3d.app"
```
Comment (Duncaen) [inline srcpkgs/f3d/template:11]: typo

Comment (Jipok) [reply srcpkgs/f3d/template]: fixed

File: srcpkgs/f3d/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+distfiles="https://github.com/f3d-app/f3d/archive/refs/tags/v${version}.tar.gz"
+checksum=3e5e6c2c16da4d7ccce8b6e316ab8007592a2bc0fc11a513f1ebac8c7f0f95d2
+
+post_install() {
+	vlicense LICENSE.md
+}
+
+do_configure() {
+	echo AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+	exit 1
```
Comment (Duncaen) [inline srcpkgs/f3d/template:22]: This looks wrong

Comment (Jipok) [reply srcpkgs/f3d/template:21]: To be honest, I rechecked this small template and did not see any problems in it. Damn, this is so stupid and embarrassing.

**Comment by Jipok** [comment]
bump

**Comment by Jipok** [comment]
`vtk >= 9.3` required for `.splat` files:
<details><summary>splat example</summary>

319067413-656b1699-e6a8-4870-9e25-624c06cd2dc4

</details>

**Comment by Jipok** [comment]
bump

**Pull request closed without merging**